README.md

```diff
@@ -31,30 +30,15 @@ Include this library as a test-scoped dependency in your POM. Ensure the `versio
 
 ```xml
 <dependency>
-  <groupId>com.deque</groupId>
-  <artifactId>axe-selenium</artifactId>
-  <version>3.0</version>
-  <scope>test</scope>
+    <groupId>com.deque.html.axe-core</groupId>
+    <artifactId>selenium</artifactId>
+    <version>3.1-SNAPSHOT</version>
+    <scope>test</scope>
 </dependency>
 ```
 
-`axe.js` or `axe.min.js` must be available to your test fixtures as a `java.net.URL`. The simplest way to do this is to include it in your own `src.test.resources` and pass `MyTest.class.getResource("/axe.min.js")` to the `Builder` constructor as demonstrated in the `ExampleTest`.
-
-The `AXE` helper defines three public methods and a nested `Builder` class for your unit tests.
-
-- `inject` will inject the required script into the page under test and any iframes. This only needs to be run against a given page once, and `Builder` will take care of it for you if you use that.
-- `report` will pretty-print a list of violations.
-- `writeResults` will write the JSON violations list out to a file with the specified name in the current working directory.
-
-The `Builder` class allows tests to chain configuration and analyze pages. The constructor takes in a `WebDriver` that has already navigated to the page under test and a `java.net.URL` pointing to the aXe script; from there, you can set `options()`, `include()` and `exclude()` selectors, `skipFrames()`, and finally, `analyze()` the page.
-
-- `options` wires a JSON string to aXe, allowing rules to be toggled on or off. See the `testAccessibilityWithOptions` unit test for a sample single-rule execution, and the [axe-core API documentation](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#b-options-parameter) for full documentation on the options object. The runOnly option with tags may be of particular interest, allowing aXe to execute all rules with the specified tag(s).
-- `include` adds to the list of included selectors. If you do not call `include` at all, aXe will run against the entire document.
-- `exclude` adds to the list of excluded selectors. Exclusions allow you to focus scope exactly where you need it, ignoring child elements you don't want to test.
-- `skipFrames` prevents aXe to be recursively injected into all iframes.
-- `analyze` executes aXe with any configuration you have previously defined. If you want to test one or more `WebElement`s, you may pass them into `analyze` instead of using `include` and `exclude`.
-
-The aXe documentation should be consulted for more details on customizing and analyzing calls to `axe.run`.
+The `AxeBuilder` type is the main interface. Pass it a Selenium `WebDriver` instance, configure it,
```

Comment:
is the `AxeBuilder` cased this way in code? If not, or if it doesn't matter then this should be `axeBuilder`

Reply:
It is. The class is named `AxeBuilder`.

Reply:
👍